tools/marvin/marvin/config/test_data.py

```diff
@@ -784,6 +784,18 @@
         "format": "OVA",
         "ispublic": "true"
     },
+    "coreos": {
+        "displaytext": "coreos",
+        "name": "coreos",
+        "passwordenabled": False,
+        "ostype": "Coreos",
+        "urlvmware":"http://10.147.28.7/templates/coreos/coreos_production_vmware.ova",
```

Comment:
cc @NuxRo I think the openvm hosted files for vmware are in wrong format (need ova instead of vmdk) which is why a local file has been used here